lib/api/repository/survey_repository.dart

```diff
@@ -13,8 +14,9 @@ abstract class SurveyRepository {
 @Singleton(as: SurveyRepository)
 class SurveyRepositoryImpl extends SurveyRepository {
   final SurveyService _surveyService;
+  final SurveyStorage _surveyStorage;
 
-  SurveyRepositoryImpl(this._surveyService);
+  SurveyRepositoryImpl(this._surveyService, this._surveyStorage);
```

Comment:
from 2 params, should we break it into multiple lines formatting?

```suggestion
  SurveyRepositoryImpl(
    this._surveyService,
    this._surveyStorage,
  );
```